sphinx/writers/latex.py

```diff
@@ -629,6 +629,8 @@ def __init__(self, document, builder):
             self.elements['contentsname'] = self.babel_renewcommand('\\contentsname',
                                                                     contentsname)
 
+        if self.config.latex_use_latexlistlabels:
+            self.elements['sphinxpkgoptions'] += ',latexlistlabels'
```

Comment:
``latexlistlabels`` serves as synonym to ``latexlistlabels=true``. The default is ``false``.